milabench/validation/validation.py

```diff
@@ -0,0 +1,128 @@
+import json
+from contextlib import contextmanager
+from dataclasses import dataclass, field
+
+from ..structs import BenchLogEntry
+
+
+class ValidationLayer:
+    """Validation layer interface, captures events, makes report"""
+
+    _rc = 0
+
+    def __init__(self, **kwargs) -> None:
+        self.early_stop = False
+        self._rc = 0
```

Comment:
I'm not sure that's a good default, I'd rather go with `None` to signify that it hasn't been set.

Reply:
done